release_notes/ocp-4-5-release-notes.adoc

```diff
@@ -1854,6 +1854,11 @@ but this is not supported. Attempting to use this option will cause the VM to
 fail to boot. Do not select this option.
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1828744[*BZ#1828744*])
 
+* There is an issue where SDN Pods in a node can lose network communication,
```

Comment:
thanks @codyhoag .  I just updated the comment as below. better also mentioned `openshift-sdn` network plugin for cluster. 
```
For use openshift-sdn networkplugin cluster. Sometimes the pods (which is using the sdn network) maybe lose the network communication. means these pods maybe cannot be working well (crashed) after the cluster is upgrade
```
`SDN Pods` may not accurate, how about using `the pods which are using sdn network`

Reply:
We've been calling openshift-sdn the _OpenShift SDN default CNI network provider_ now.

Reply:
Thanks, @zhaozhanqi @jboxman. I updated according to your feedback.